seqr/utils/search/utils.py

```diff
@@ -32,6 +35,94 @@ def get_es_client(timeout=60, **kwargs):
     return elasticsearch.Elasticsearch(**client_kwargs, **kwargs)
 
 
+def ping_search_backend():
+    if not get_es_client(timeout=3, max_retries=0).ping():
+        raise ValueError('No response from elasticsearch ping')
+
+
+def get_elasticsearch_status():
```

Comment:
copy-pasted from data_manager_api

Reply:
The better file location for the non-search database management operations (e.g., status checking, loading, deleting, etc.) should be under the database-specific folder because they are not general to all types of databases. But here is the right place if we have a general DB status checking for all DBs.

Reply:
Thats a fair point, I can give this a more generic name and then move all the ES specific logic into the elasticsearch helper function